README.md

```diff
@@ -1 +1,195 @@
+
+   #### Fetch and run the official installer
+
+   ```bash
+   curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
+   ```
+
+   #### In your shell (or reopen it):
+
+   ```bash
+   source ~/.cargo/env
+   ```
+
+   #### Make sure you’re up to date
+
+   ```bash
+   rustup update
+   ```
+
+3.  [Tauri Prerequisites](https://tauri.app/v1/guides/getting-started/prerequisites) (Ensure you follow the setup for your specific Operating System, including mobile development requirements if applicable).
+
+    Tauri needs some native toolkits to build its WebView.
+
+    #### macOS
+
+    ##### Xcode Command-Line Tools
+
+    ```bash
+    xcode-select --install
+    ```
+
+    #### Ubuntu / Debian
+
+    ```bash
+    sudo apt install -y build-essential pkg-config libgtk-3-dev libwebkit2gtk-4.0-dev
+    ```
+
+    #### Fedora / RHEL
+    ```bash
+    sudo dnf install -y gcc-c++ make pkgconfig gtk3-devel webkit2gtk3-devel
+    ```
+
+    #### Windows
+
+    1.  **Visual Studio 2022 Build Tools**
+        - Download & install the “Desktop development with C++” workload (include Windows 10+ SDK).
+    2.  **Windows SDK** (if not bundled)
+    3.  **(Optional) Python 2** for some Node native modules:
+        ```bash
+        choco install python2 -y
+        ```
+    4.  Ensure your toolchain target is set:
+        ```bash
+        rustup default stable-x86_64-pc-windows-msvc
+        ```
+
+
+### Enviroment setup
+
+1.  **Clone the repository:**
+    ```bash
+    git clone https://github.com/corpora-inc/yijing.git
+    cd yijing
+    ```
+2.  **Install frontend dependencies:**
+    ```bash
+    npm install
+    ```
+3.  **Run in development mode (Desktop):**
+    ```bash
+    npm run tauri dev
+    ```
+    This will open the app in a development window with hot-reloading for the frontend.
+4.  **Build the application (Desktop):**
+    ```bash
+    npm run tauri build
+    ```
+    Find the installers/binaries in `src-tauri/target/release/bundle/`.
+5.  **Mobile Development:**
+    - Refer to the official [Tauri Mobile documentation](https://tauri.app/v1/guides/mobile/) for detailed instructions on setting up emulators/simulators and running/building for iOS and Android.
+    - Common commands: `npm run tauri android dev`, `npm run tauri ios dev`, `npm run tauri android build`, `npm run tauri ios build`.
+
+#### Tips & gotchas
+
+- **Updating Tauri:** upgrade both the CLI and the core crates in lockstep:
+  cargo install tauri-cli --force
+  npm install @tauri-apps/cli@latest
+- **Troubleshooting:**
+  - On macOS, if the build fails in `cocoa`, make sure Xcode tools are current.
+  - On Windows, double-check that the MSVC toolchain is active (`rustup show`).
+
+## 🤝 Contributing
+
+Contributions are what make the open-source community such an amazing place to learn, inspire, and create. Any contributions you make are **greatly appreciated**.
+
+1.  Fork the Project
+2.  Create your Feature Branch (`git checkout -b feature/AmazingFeature`)
```

Comment:
I don't personally follow any `feature/` convention ... I'm a free spirit when it comes to my code. I wrote this a long time ago (never finished) - https://skyl.github.io/bonsai/ - maybe I will use corpora to rewrite and finish it :). I have a very free-wheeling approach to process though ...